forms-flow-web/src/components/ServiceFlow/details/TaskHeader.js

```diff
@@ -163,99 +160,121 @@ const TaskHeader = React.memo(() => {
       />
       <Row className="mx-0 task-header">{task?.name}</Row>
       <Row className="mx-0 task-name">
-        <span className="application-id" 
-         title={t("Process Name")} 
-         data-bs-toggle="tooltip"  
-         >
-          {" "}
-          {
-            getProcessDataObjectFromList(processList, task?.processDefinitionId)
-              ?.name
-          }
+        <span className="application-id">
+          <OverlayTrigger
```

Comment:
can you make the overlay Trigger as higher-order component to reduces the code redundancy

Reply:
I think this is not necessary because if we give a title then automatically tool tip will show. so here I am expecting remove the style form style sheet